packages/app-store/exchange2016calendar/_metadata.ts

```diff
@@ -0,0 +1,26 @@
+import type { App } from "@calcom/types/App";
+
+import _package from "./package.json";
+
+export const metadata = {
+  name: "Microsoft Exchange 2016 Calendar",
+  description: _package.description,
+  installed: true,
+  type: "exchange2016_calendar",
+  title: "Microsoft Exchange 2016 Calendar",
+  imageSrc: "/api/app-store/exchange2016calendar/icon.svg",
+  variant: "calendar",
+  category: "calendar",
+  label: "Exchange Calendar",
+  logo: "/api/app-store/exchange2016calendar/icon.svg",
+  publisher: "Cal.com",
+  rating: 5,
+  reviews: 69,
+  slug: "exchange2016-calendar",
+  trending: false,
+  url: "https://cal.com/",
```

Comment:
Be aware of @alannnc changed introducing a new flag to allow multiple installations for an app, which applies to these new Exchange apps: #3045

Reply:
Thanks for the heads up. I'll wait for the PR to be merged to add this to my PR.

Reply:
This won't be necessary. Metadata will be deprecated so all new changes should be done in the DB.